src/app/landing-page/create-avatar/create-avatar.component.ts

```diff
@@ -34,6 +35,7 @@ export class CreateAvatarComponent {
   st = inject(StorageService);
   user = inject(UserServiceService);
   router = inject(Router);
+  stateControl = inject(StateControlService)
```

Comment:
Either use or skip using `;` (across all files) or use Prettier or other tool to add/strip the semicolons